docs/apache-airflow/faq.rst

```diff
@@ -398,7 +398,7 @@ What does ``TemplateNotFound`` mean?
 -------------------------------------
 
 ``TemplateNotFound`` errors are usually due to misalignment with user expectations when passing path to operator
-that trigger Jinja templating. A common occurrence is with :ref:`BashOperators<howto/operator:BashOperator>`.
+that trigger Jinja templating. A common occurrence is with `BashOperators <https://airflow.apache.org/docs/apache-airflow-providers-standard/stable/operators/bash.html>`__.
```

Comment:
i don't think http links are good for internal references.  This can easily be broken.
If we can't use the `:ref:` between packages lets use `:class:`

example for cross package reference
https://github.com/apache/airflow/blob/857ca4c06c9008593674cabdd28d3c30e3e7f97b/docs/apache-airflow-providers-mysql/operators.rst#L25

Reply:
make sense , thanks for the suggestion :)